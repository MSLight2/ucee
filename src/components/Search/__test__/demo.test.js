import { demoTest } from '../../../tests/shared/demoTest';
import SearchDemo from '../demo';

demoTest('Search', SearchDemo);