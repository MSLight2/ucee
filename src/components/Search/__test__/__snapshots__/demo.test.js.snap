// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search demo test 1`] = `
Array [
  <div
    className="ucee-search"
    style={
      Object {
        "background": undefined,
      }
    }
  >
    <div
      className="ucee-search__content"
    >
      <input
        className="ucee-search__input"
        onChange={[Function]}
        onKeyDown={[Function]}
        placeholder="请输入搜索关键字"
        type="search"
        value=""
      />
    </div>
    <div
      className="ucee-search__action"
    />
  </div>,
  <div
    className="ucee-search"
    style={
      Object {
        "background": undefined,
      }
    }
  >
    <div
      className="ucee-search__content"
    >
      <span
        className="ucee-search__lable"
      >
        地址
      </span>
      <input
        className="ucee-search__input"
        onChange={[Function]}
        onKeyDown={[Function]}
        placeholder="请输入搜索关键字"
        type="search"
        value=""
      />
    </div>
    <div
      className="ucee-search__action"
    />
  </div>,
  <div
    className="ucee-search"
    style={
      Object {
        "background": undefined,
      }
    }
  >
    <div
      className="ucee-search__content"
    >
      <span
        className="ucee-search__lable"
      >
        清除按钮
      </span>
      <input
        className="ucee-search__input ucee-search__input-clear"
        onChange={[Function]}
        onKeyDown={[Function]}
        placeholder="请输入搜索关键字"
        type="search"
        value=""
      />
    </div>
    <div
      className="ucee-search__action"
    />
  </div>,
  <div
    className="ucee-search"
    style={
      Object {
        "background": undefined,
      }
    }
  >
    <div
      className="ucee-search__content"
    >
      <span
        className="ucee-search__lable"
      >
        地址
      </span>
      <input
        className="ucee-search__input"
        onChange={[Function]}
        onKeyDown={[Function]}
        placeholder="请输入搜索关键字"
        readOnly={true}
        type="search"
        value=""
      />
    </div>
    <div
      className="ucee-search__action"
    />
  </div>,
  <div
    className="ucee-search"
    style={
      Object {
        "background": undefined,
      }
    }
  >
    <div
      className="ucee-search__content ucee-search--disable"
    >
      <input
        className="ucee-search__input"
        disabled={true}
        onChange={[Function]}
        onKeyDown={[Function]}
        placeholder="disabled"
        type="search"
        value=""
      />
    </div>
    <div
      className="ucee-search__action"
    />
  </div>,
  <div
    className="ucee-search"
    style={
      Object {
        "background": undefined,
      }
    }
  >
    <div
      className="ucee-search__content"
    >
      <input
        className="ucee-search__input"
        onChange={[Function]}
        onKeyDown={[Function]}
        placeholder="请输入搜索关键字"
        type="search"
        value=""
      />
    </div>
    <div
      className="ucee-search__action"
      onClick={[Function]}
    >
      取消
    </div>
  </div>,
  <div
    className="ucee-search"
    style={
      Object {
        "background": undefined,
      }
    }
  >
    <div
      className="ucee-search__content"
    >
      <input
        className="ucee-search__input"
        onChange={[Function]}
        onKeyDown={[Function]}
        placeholder="请输入搜索关键字"
        style={
          Object {
            "textAlign": "center",
          }
        }
        type="search"
        value=""
      />
    </div>
    <div
      className="ucee-search__action"
    />
  </div>,
  <div
    className="ucee-search"
    style={
      Object {
        "background": undefined,
      }
    }
  >
    <div
      className="ucee-search__content ucee-search--round"
    >
      <input
        className="ucee-search__input"
        onChange={[Function]}
        onKeyDown={[Function]}
        placeholder="半圆框形状"
        type="search"
        value=""
      />
    </div>
    <div
      className="ucee-search__action"
      onClick={[Function]}
    >
      取消
    </div>
  </div>,
  <div
    className="ucee-search"
    style={
      Object {
        "background": undefined,
      }
    }
  >
    <div
      className="ucee-search__content"
    >
      <input
        className="ucee-search__input"
        onChange={[Function]}
        onKeyDown={[Function]}
        placeholder="请输入搜索关键字"
        type="search"
        value=""
      />
    </div>
    <div
      className="ucee-search__action"
      onClick={[Function]}
    >
      自定义文字
    </div>
  </div>,
  <div
    className="ucee-search"
    style={
      Object {
        "background": undefined,
      }
    }
  >
    <div
      className="ucee-search__content"
    >
      <input
        className="ucee-search__input"
        onChange={[Function]}
        onKeyDown={[Function]}
        placeholder="请输入搜索关键字"
        type="search"
        value=""
      />
    </div>
    <div
      className="ucee-search__action"
    >
      <button
        className="ucee-button ucee-button--primary ucee-button--mini"
        onClick={[Function]}
        style={Object {}}
      >
        搜索
      </button>
    </div>
  </div>,
  <div
    className="ucee-search"
    style={
      Object {
        "background": undefined,
      }
    }
  >
    <div
      className="ucee-search__content"
    >
      <i
        className="ucee-icon ucee-icon-search"
        style={Object {}}
      />
      <input
        className="ucee-search__input"
        onChange={[Function]}
        onKeyDown={[Function]}
        placeholder="请输入搜索关键字"
        type="search"
        value=""
      />
    </div>
    <div
      className="ucee-search__action"
    />
  </div>,
  <div
    className="ucee-search"
    style={
      Object {
        "background": undefined,
      }
    }
  >
    <div
      className="ucee-search__content"
    >
      <input
        className="ucee-search__input"
        onChange={[Function]}
        onKeyDown={[Function]}
        placeholder="请输入搜索关键字"
        type="search"
        value=""
      />
      <i
        className="ucee-icon ucee-icon-search"
        style={Object {}}
      />
    </div>
    <div
      className="ucee-search__action"
    />
  </div>,
  <div
    className="ucee-search"
    style={
      Object {
        "background": "red",
      }
    }
  >
    <div
      className="ucee-search__content"
    >
      <input
        className="ucee-search__input"
        onChange={[Function]}
        onKeyDown={[Function]}
        placeholder="请输入搜索关键字"
        type="search"
        value=""
      />
    </div>
    <div
      className="ucee-search__action"
    />
  </div>,
  <div
    className="ucee-search"
    style={
      Object {
        "background": undefined,
      }
    }
  >
    <div
      className="ucee-search__content"
    >
      <input
        className="ucee-search__input"
        onChange={[Function]}
        onKeyDown={[Function]}
        placeholder="请输入搜索关键字"
        type="search"
        value=""
      />
    </div>
    <div
      className="ucee-search__action"
    />
  </div>,
]
`;
